### DISCUSSION TITLE
How can I remove this ghostly opening tx?

### DISCUSSION THREAD

**Discussion by LNOrville** [lightningnetwork/lnd#7179] [Troubleshooting] [answered]
**Background**
Unknown node with ID 03e3ef3b9945fd025e09cefd53fded3373c9b81da2ac63eadba760d328c18a15ff initiated opening of a channel with my node. The opening tx cannot be found in mempool. The opening node cannot be found in any lightning explorers and attempts to peer with the node outputs “Unable To Find Node”.
My node sees the channel opening as stuck in pending open. This tx is going nowhere and I would like to remove it from my node.

**Your environment**

LND 0.15.4-beta
Umbrel on a RasPi

**Steps to reproduce** 
The opening was initiated by the unknown node and I did not do anything actively for this to happen, so I don’t know how to reproduce it.

Maybe the unknown node initiated opening with a low mempool fee while mempool was full (as has been the case for some days now), and the transaction was purged from mempool?

This is the situation:

 "pending_open_channels": [
        {
            "channel": {
                "remote_node_pub": "03e3ef3b9945fd025e09cefd53fded3373c9b81da2ac63eadba760d328c18a15ff",
                "channel_point": "9047224fb415c9a144f9c634db9211c60ef6e7d07db8221a5a9af50766ad9563:0",
                "capacity": "300000",
                "local_balance": "0",
                "remote_balance": "296530",
                "local_chan_reserve_sat": "3000",
                "remote_chan_reserve_sat": "3000",
                "initiator": "INITIATOR_REMOTE",
                "commitment_type": "ANCHORS",
                "num_forwarding_packages": "0",
                "chan_status_flags": ""
            },
            "confirmation_height": 0,
            "commit_fee": "2810",
            "commit_weight": "772",
            "fee_per_kw": "2500"

**Expected behaviour**
I guess this tx will stay in my node as pending open until I can actively remove it, so **I hope someone here can tell me how to do so.**

**Comment by yyforyongyu** [comment, marked as answer]
There is the command `abandonchannel`. However, I recommend **waiting for a few days before taking any action** just in case it eventually gets through as the mempool is congested these days.

**Reply by LNOrville** [reply]
Input: lncli abandonchannel 9047224fb415c9a144f9c634db9211c60ef6e7d07db8221a5a9af50766ad9563 0 
Output: [lncli] rpc error: code = Unknown desc = AbandonChannel RPC call only available in dev builds

**Reply by LNOrville** [reply]
#7151 looks like a similar issue - but in this case I am not able to peer with the the unknown opening node, and it cannot be found in any explorers either. So this opening attempt will be forgotten after 14 days? We shall see :-)

**Reply by guggero** [reply]
Check `lncli abandonchannel`, there's a parameter to overwrite the "dev builds only" behavior. But the channel just confirmed, so DO NOT abandon it or you will lose the funds.

**Reply by LNOrville** [reply]
Thank you! Unknown node force closed the channel before any routing happened.
